### DISCUSSION TITLE
How to make pluralization API give me `n(count)` instead of raw `count`?

### DISCUSSION THREAD

**Discussion by Shayan-To** [intlify/vue-i18n#912] [Q&A]
In `fa` locale, we use localized digits. I tried using the pluralization API and added `days: "{count} روز"` to my messages, but `t("days", 5)` gives me the raw `count` (`5 روز` instead of `۵ روز`).

What is the correct way of doing that? (I prefer my usage site, where I call `t`, to remain simple if possible.)

**Comment by kazupon** [comment]
You might need the custom pluralization.
In about it, docs said:
https://vue-i18n.intlify.dev/guide/essentials/pluralization.html#custom-pluralization
